jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -81,10 +81,10 @@ $z-indexed-elements: search-results-list-block__tags,
     search-results-cell--hover,
     boolean-config--handle,
     boolean-config--label,
+    inspector-view,
```

Comment:
Yea, I had to change the layer logic here so when you load machine view it appears inferring of the inspector. Saves having to unload and reload the inspector in switch. 

Smoke and mirrors on small screens